README.md

```diff
@@ -1,13 +1,13 @@
+# Therapist
+Work out your commitment issues. A smart pre-commit hook for git.
```

Comment:
This would be good on a t-shirt.

Reply:
Maybe I'll get some for the next all hands

Reply:
It'd be great on stickers, too. Plus they're cheaper.